src/pymovements/dataset/dataset.py

```diff
@@ -1057,6 +1057,14 @@ def extract(
         )
         return self
 
+    @property
+    def info(self) -> None:
+        """The information about the dataset.
+
+        Print dataset information and citation key.
```

Comment:
the property should return a string instead of printing it. A user can easily call `print(dataset.info)` if necessary